packages/js/src/dashboard/widgets/site-kit-setup-widget.js

```diff
@@ -44,6 +45,7 @@ const useSiteKitConfiguration = ( dataProvider, remoteDataProvider ) => {
 		).then( ( { success } ) => {
 			if ( success ) {
 				dataProvider.setSiteKitConnected( true );
+				addWidget( "topPages", 0 );
```

Comment:
Kind of feels weird to have this random string here, since it is linked to another widget. I would love this to be a constant that lives in the widget it adds here.